generator/graphql-kotlin-federation/src/test/kotlin/com/expediagroup/graphql/generator/federation/data/TestResolvers.kt

```diff
@@ -27,7 +27,7 @@ import java.util.concurrent.CompletableFuture
 internal class BookResolver : FederatedTypeSuspendResolver<Book> {
     override val typeName: String = "Book"
 
-    override suspend fun resolve(environment: DataFetchingEnvironment, representation: Map<String, Any>): Book? {
+    override suspend fun resolve(environment: DataFetchingEnvironment, representation: Map<String, Any>): Book {
```

Comment:
nit: technically the implementations here always return objects but in the real life examples it should be nullable returns

Reply:
just did this to keep the IDE happy.